### DISCUSSION TITLE
diesel_cli: add an option to skip generating 00000000000000_diesel_initial_setup in database setup

### DISCUSSION THREAD

**Discussion by Zstorm999** [diesel-rs/diesel#4148] [Ideas]
Hello !

As the title says, it would be nice to have an option to skip generating this file if it is not needed. 

It could be something like `diesel database setup --no-default-migration`.

**Comment by weiznich** [comment]
I'm happy to accept a PR for that, as long as it includes a test + documentation.

**Comment by lgrosz** [comment]
[I took at stab at this](https://github.com/diesel-rs/diesel/pull/4184)...
